charts/dbildungscloud-iam-keycloak/templates/ingress.yaml

```diff
@@ -1,20 +1,22 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: keycloak-ingress
```

Comment:
Please add a namespace.

Reply:
More precisely, .Release.Namespace, however I seem to recall that helm actually does this on its own?